src/api/emails/application/get-submitted-confirmation-template-id/index.ts

```diff
@@ -15,33 +17,52 @@ const {
 } = EMAIL_TEMPLATE_IDS;
 
 /**
- * getSubmittedConfirmationTemplateId.submittedEmail
+ * getSubmittedConfirmationTemplateId
  * Get an email template ID for the "application submitted" email, depending on:
  * - The application's policy type.
- * - If the "maximum buyer will owe" is below a threshold.
+ * - If the "maximum buyer will owe" (in GBP) is below a threshold.
  * @param {ApplicationPolicy} policy: Application policy
- * @returns {String} callNotify response
+ * @returns {Promise<String>} "application submitted" template ID
  */
-const getSubmittedConfirmationTemplateId = (policy: ApplicationPolicy): string => {
-  const { policyType, maximumBuyerWillOwe } = policy;
+const getSubmittedConfirmationTemplateId = async (policy: ApplicationPolicy): Promise<string> => {
+  try {
+    console.info('Getting submitted confirmation template ID (getSubmittedConfirmationTemplateId helper)');
 
-  if (isSinglePolicyType(policyType)) {
-    return CONFIRMATION.SINGLE_OR_MULTIPLE_CONTRACT_POLICY;
-  }
+    const { maximumBuyerWillOwe, policyCurrencyCode, policyType } = policy;
+
+    if (isSinglePolicyType(policyType)) {
+      return CONFIRMATION.SINGLE_OR_MULTIPLE_CONTRACT_POLICY;
+    }
+
+    if (isMultiplePolicyType(policyType) && maximumBuyerWillOwe) {
+      let maximumBuyerWillOweInGbp = maximumBuyerWillOwe;
+
+      if (policyCurrencyCode !== GBP) {
+        const source = GBP;
+        const target = String(policyCurrencyCode);
 
-  if (isMultiplePolicyType(policyType) && maximumBuyerWillOwe) {
-    const threshold = Number(SMALL_EXPORT_BUILDER?.MAXIMUM_BUYER_WILL_OWE);
+        const exchangeRate = await apimCurrencyExchangeRate.get(source, target);
 
-    const eligibileForSmallExportBuilder = maximumBuyerWillOwe <= threshold;
+        maximumBuyerWillOweInGbp = roundNumber(maximumBuyerWillOwe * exchangeRate);
+      }
 
-    if (eligibileForSmallExportBuilder) {
-      return CONFIRMATION.MULTIPLE_CONTRACT_POLICY.ELIGIBLE_FOR_SMALL_EXPORT_BUILDER_CONFIRMATION;
+      const threshold = Number(SMALL_EXPORT_BUILDER?.MAXIMUM_BUYER_WILL_OWE);
+
+      const eligibileForSmallExportBuilder = maximumBuyerWillOweInGbp <= threshold;
+
+      if (eligibileForSmallExportBuilder) {
+        return CONFIRMATION.MULTIPLE_CONTRACT_POLICY.ELIGIBLE_FOR_SMALL_EXPORT_BUILDER_CONFIRMATION;
+      }
+
+      return CONFIRMATION.SINGLE_OR_MULTIPLE_CONTRACT_POLICY;
     }
 
-    return CONFIRMATION.SINGLE_OR_MULTIPLE_CONTRACT_POLICY;
-  }
+    return '';
```

Comment:
could this be something else instead of an empty string?

Reply:
If we do that we'd need to have 2x return types e.g `String | null`, I think one type with an empty string is cleaner.

In any instance, the Notify call will fail due to unrecognised template ID.

Thoughts?

Reply:
I'm updating this to return a new constant, `UNABLE_TO_DETERMINE_TEMPLATE_ID` which would be very helpful for debugging.

Reply:
I think a new constant would be better than an empty string